### PR TITLE
gh #229 VTS - L1 Issue fix - SoC Vendor Issues - dsVideoPort 

### LINKED ISSUE #229: VTS L1-  Issues - dsVideoPort
Issues found on release [3.1.2](https://github.com/rdkcentral/rdk-halif-test-device_settings/milestone/4)

### Issues Reported ###
1. dsGetColorDepth_L1_Positive - Compare the Color Depth read correctly.
2. GetHdmiPreference_L1_negative  - Validate the result at step 5 with dsERR_INVALID_PARAM

### PR DESCRIPTION
**Pull request by akhilbhas**
* fix for the issues reported
Closes #229

### CODE REVIEW

File: src/test_l1_dsVideoPort.c [hunk 1]
```diff
@@ -3200,7 +3200,7 @@ void test_l1_dsVideoPort_positive_dsGetColorDepth(void) {
         status = dsGetColorDepth(handle, &colorDepth1);
         UT_ASSERT_EQUAL(status, dsERR_NONE);
         // Step 04: Compare the value with values from profile file
-        UT_ASSERT_EQUAL(colorDepth1, gDSvideoPort_color_depth);
+        UT_ASSERT_EQUAL(colorDepth1, getColorDepth());
```
Comment (hari22yuva) [inline src/test_l1_dsVideoPort.c:3203]: rather this we need update the profile with color_depth number 
if you see the https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/main/profiles/source/Source_4K_VideoPort.yaml#L13

Comment (srinivasgtl) [reply src/test_l1_dsVideoPort.c:3203]: Can you consider this to update the right value in profile and suggest the same with MTK guys on jira.

Comment (akhilbhas) [reply src/test_l1_dsVideoPort.c]: done

Comment (hari22yuva) [reply src/test_l1_dsVideoPort.c]: i didn't see any change in profile file

**Comment by hari22yuva** [comment]
> * fix for the issues reported
>   Closes [VTS L1-  SoC Vendor Issues - dsVideoPort   #229](https://github.com/rdkcentral/rdk-halif-test-device_settings/issues/229)

Can you explain what is issue

File: src/test_l1_dsVideoPort.c (hunk 1 above)
Comment (akhilbhas) [reply src/test_l1_dsVideoPort.c]: https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/9723390ddffea922ec08c399cf0998b3cccc4db1/profiles/sink/Sink_4K_VideoPort.yaml#L13

Updated value in sink to 8  (dsDISPLAY_COLORDEPTH_AUTO)

**Pull request merged**